### ISSUE TITLE
Problem with the dots in "Office" programs 

### ISSUE THREAD

**Issue by Deksteroo** [jpt/barlow#72]
Hello,

I am reporting a problem with the font in MS Office and LibreOffice.

I get an error and strange vertical dots when inserting multiple dots and printing my document.
Pictures below.
![1](https://user-images.githubusercontent.com/81414371/112618906-f11c5780-8e26-11eb-9722-8f57dde930d8.PNG)
![2](https://user-images.githubusercontent.com/81414371/112618907-f24d8480-8e26-11eb-9df6-cb9698cd32a7.PNG)
![3](https://user-images.githubusercontent.com/81414371/112618909-f24d8480-8e26-11eb-9848-0aef4cc97677.PNG)

The problem occurs when printing directly from "Word" programs, printing from a saved PDF file is correct.

I also noticed that the dots change every 3. As in the picture below.
![A1](https://user-images.githubusercontent.com/81414371/112618923-fa0d2900-8e26-11eb-8abd-9432cc15e6c2.png)

Can I ask for help with the topic?

**Comment by Deksteroo** [comment]
I found out that the problem is font interpretation by the PCL-6 printer driver.
In the print quality settings for Vector, printing is incorrect when Raster is good.
Setting a permanent print as Raster is not a solution because the quality of the prints drops ...
any other solution?

**Comment by agusmba** [comment]
Shot in the dark here.
I found that previous versions of the font had problems with some of our printers depending on whether I was using the ttf or the otf variants. You could try uninstalling the font and installing the other format just in case...

What version of the font are you using? (1.422 ttf is working fine here)

**Comment by kenmcd** [comment]
> 
> 
> Shot in the dark here.
> I found that previous versions of the font had problems with some of our printers depending on whether I was using the ttf or the otf variants. You could try uninstalling the font and installing the other format just in case...
> 
> What version of the font are you using? (1.422 ttf is working fine here)

Are you sure it is the TTF version which works, and the OTF which does not?

Why I ask ... 
HP PCL printer drivers have a problem with nested components within a font.
We have seen this issue with stacked diacritics (accents) which are nested components within a font.
The accents end up in odd places when printed directly through the printer driver.
Printing it to the PDF first avoids this because the PDF "printer driver" does not have this issue.

I think what is happening here is that Word is replacing each three dots `...` with the ellipsis character.
The ellipsis character in the OTF font is three separate contours (no nested components)
But the ellipsis in the TTF font is three separate components (3 copies of the period).
And I think these are getting stacked when printed - which is why you see the stacks of three dots.

If the TTF font is the one that is working - then my theory has an issue. :-)

**Comment by agusmba** [comment]
> Are you sure it is the TTF version which works, and the OTF which does not?

No, I don't recall wich variant gave me problems, and since then we have changed the printers at work (now ttf are working, but I think otf also work here). Sorry for the noise.

**Comment by Deksteroo** [comment]
We tested several versions of the TTF font. There are errors on each of them.

The error is not even the same. On various printers, sometimes they are dots, as I wrote above, and sometimes they are smaller and more dense.
![TA](https://user-images.githubusercontent.com/81414371/120986808-435deb00-c77d-11eb-87a8-26cd3f505850.PNG)

From what we noticed, the errors concern only Konica Minolta printers.
Unfortunately, the printer service also does not know the reason ...

Maybe someone knows where to report such a problem ...

**Comment by kenmcd** [comment]
> We tested several versions of the TTF font. There are errors on each of them.

The fonts are actually OK.  
The issue is the printer driver does not properly handle the use of components in OpenType-TT (TTF) fonts.
One solution, or work-around, is to use the OpenType-PS (OTF) fonts - which do not use components.
Another option is to remove the use of components in the TTF fonts (which will make the file size slightly larger).
I can make you a version of the TTF fonts with the components removed if you would like to test.

**Comment by Deksteroo** [comment]
Yes please, it would be great!

**Comment by jpt** [comment]
I will remove the components in the ellipses as a build step. Thanks for flagging this.

**Comment by Deksteroo** [comment]
This is great news! 
Thank you and I look forward to the new release!

**Comment by jpt** [comment]
there are new TTF files here, let me know how they work for you if you get a chance https://github.com/jpt/barlow/tree/1.5/fonts/ttf

still testing it myself

**Comment by Deksteroo** [comment]
We are still testing but it seems that the font now prints correctly. Thank you !
